packages/app/src/files-manager/components/files-manager/files-manager.component.tsx

```diff
@@ -62,6 +62,7 @@ const FilesManager: FunctionComponent<FilesManagerProps> = ({
   abortPendingUpload,
   continuePendingUpload,
 }) => {
+  const uploadTimeoutRef = useRef<ReturnType<typeof setTimeout> | null>(null)
```

Comment:
consider default value to be 'undefined' and when clearing timeout "|| undefined" won't be necessary :)